x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/reducer.ts

```diff
@@ -443,6 +472,11 @@ export function reducer(state: State, action: Action): State {
         newFormState.sourceIndexNameValid = Object.keys(validationMessages).length === 0;
       }
 
+      if (action.payload.numTopFeatureImportanceValues !== undefined) {
+        newFormState.numTopFeatureImportanceValuesValid =
+          (newFormState?.numTopFeatureImportanceValues ?? 0) > 0;
```

Comment:
Shouldn't this be `>= 0` ? `num_top_feature_importance_values` of `0` is valid if you don't want feature importance calculation.

Reply:
Thanks for spotting this, fixed form validation in b4e2b69.